docs/advanced.rst

```diff
@@ -521,7 +521,7 @@ Next we need to define reference. Inside Model_Invoice add::
         $m->ref('InvoicePayment')->action('delete')->execute();
 
         // If you have important per-row hooks in InvoicePayment
-        // $m->ref('InvoicePayment')->each('delete');
+        // $inv = $m->ref('InvoicePayment'); $inv->each(function () use ($inv) { $inv->delete(); });
```

Comment:
Should be 
`$payment = $m->ref('InvoicePayment'); $payment->each(function () use ($payment) { $payment->delete(); });`

Reply:
done